src/models/ChatConfig.ts

```diff
@@ -19,10 +19,10 @@ export interface ChatConfig {
    * The version of the chat bot configuration.
    *
    * @remarks
-   * Default to 'STAGING' in Chat API
+   * Default to 'LATEST' in Chat API
```

Comment:
can you run `npm run build` to re-generate the docs?
Also publish a version for this change

Reply:
done